client/src/game/shapes/shape.ts

```diff
@@ -339,6 +360,16 @@ export abstract class Shape {
         if (sync) socket.emit("Shape.Layer.Change", { uuid: this.uuid, layer, floor: newLayer.floor });
     }
 
+    rotate(degrees: number, sync: boolean): void {
+        const oldLayer = layerManager.getLayer(this.floor, this.layer);
```

Comment:
Why is it called 'oldLayer' ? the layer doesn't change so the naming might seem confusing.

Reply:
Copied and pasted from the Shape.moveFloor() and didn't rename it, will fix.

Reply:
No longer used, swapped to this.invalidate(true)